cmd/ttn-lw-cli/commands/use.go

```diff
@@ -53,6 +70,11 @@ var (
 			overwrite, _ := cmd.Flags().GetBool("overwrite")
 
 			host := args[0]
+			if !validHostnameRegex.MatchString(host) {
+				logger.Warn("Specify the hostname only, for example `eu1.cloud.thethings.network`")
```

Comment:
I don't think we use backticks in log messages

Reply:
This is not addressed, please change or comment here

Reply:
You don't need to log a warning if you also return an error.